### ISSUE TITLE
installation BERTopic & umap

### ISSUE THREAD

**Issue by aph61** [MaartenGr/BERTopic#232]
Hi,

I made a fresh virtual environment, but I keep having problems with importing BERTopic

from bertopic import BERTopic

/usr/local/lib/python3.8/dist-packages/bertopic/__init__.py in <module>
----> 1 from bertopic._bertopic import BERTopic
      2 
      3 __version__ = "0.9.0"
      4 
      5 __all__ = [
ImportError: cannot import name 'UMAP' from 'umap' (/usr/local/lib/python3.8/dist-packages/umap/__init__.py)

Any solution? There are many having this problem

thanks,

Andreas

**Comment by MaartenGr** [comment]
When creating a virtual environment, did you make sure no packages were installed in that environment? The ImportError that you experience seems to be related to how you installed UMAP. Did you install BERTopic and UMAP through `pip install bertopic`?

**Comment by aph61** [comment]
Hi Maarten,

thanks for the fast response. I made a couple of errors when installing the packages, this being the largest

- I installed them as admin (sudo pip3 install bertopic). That gives you dist-packages, not site-packages

I uninstalled everything as root, removed manually all files/directories, and re-installed the files in the normal environment (not in the virtual environment). After that the imports worked both in the notebook instantiated from the normal environment, and the virtual environment. 

So it was a series of installation errors on my part. Thanks for the pointers

best, Andreas
